docs/reference-metrics.md

```diff
@@ -113,7 +113,7 @@ in the `bookkeeper.conf` configuration file.
 
 ## Broker
 
-The broker metrics are exposed under "/metrics" at port `8080`. You can change the port by updating `webServicePort` to a different port
```

Comment:
Is there some difference between them? In HTTP protocol, the `/metrics` is equal to `/metrics/`

Reply:
Hi @hangc0276 can you explain this?

Reply:
Hi @horizonzy as we discussed (+@hangc0276) just now, users can not get data if using `/metrics` with `curl`, they need to use `/metrics/`.

(A workaround is to add `-L` to redirect, but we do not add this workaround to avoid making docs complicated)

Can you approve this PR if no more comments? Thank you!

Reply:
Get it